assets/js/components/OptIn.js

```diff
@@ -92,29 +92,22 @@ export default function OptIn( {
 				loading={ enabled === undefined }
 				alignLeft={ alignLeftCheckbox }
 			>
-				<span>
-					{ __(
-						'Help us improve Site Kit by sharing anonymous usage data.',
+				{ createInterpolateElement(
+					__(
+						'<span>Help us improve Site Kit by sharing anonymous usage data. </span><span>All collected data is treated in accordance with the <a>Google Privacy Policy.</a></span>',
```

Comment:
I believe this is how it should be done, no?
```suggestion
						'<span>Help us improve Site Kit by sharing anonymous usage data.</span> <span>All collected data is treated in accordance with the <a>Google Privacy Policy.</a></span>',
```